ReportingServicesTools/Functions/Admin/Set-RsDatabase.ps1

```diff
@@ -7,52 +7,76 @@ function Set-RsDatabase
     <#
         .SYNOPSIS
             This script configures the database settings used by SQL Server Reporting Services.
-        
+
         .DESCRIPTION
             This script configures SQL Server Reporting Services to either create and use a new RS database or use an existing RS database.
             You must be an admin in RS and SQL Server in order to perform this operation successfully.
-        
+            There are three phases to setup:
+            1. Create the PBIRS database on the database server
+            2. Create the run-time user
+            3. Grant the run-time user access to the PBIRS database
+            4. Configure the PowerBI Report Server to use the database and run-time credentials
+            Your admin role on SQL Server can be one of:
+            * The account under which you run this powershell (default)
+            * A specific set of credentials for the SQL Server which has admin rights, specified via -AdminDatabaseCredential
+
         .PARAMETER ReportServerInstance
             Specify the name of the SQL Server Reporting Services Instance.
             Use the "Connect-RsReportServer" function to set/update a default value.
-        
+
         .PARAMETER ReportServerVersion
             Specify the version of the SQL Server Reporting Services Instance.
             Use the "Connect-RsReportServer" function to set/update a default value.
-        
+
         .PARAMETER ComputerName
             The Report Server to target.
             Use the "Connect-RsReportServer" function to set/update a default value.
-        
+
         .PARAMETER Credential
             Specify the credentials to use when connecting to the Report Server.
             Use the "Connect-RsReportServer" function to set/update a default value.
-        
+
         .PARAMETER DatabaseServerName
             Specify the database server name. (e.g. localhost, MyMachine\Sql2016, etc.)
-        
+
         .PARAMETER IsRemoteDatabaseServer
             Specify this switch if the database server is on a different machine than the machine Reporting Services is running on.
-        
+
         .PARAMETER Name
             Specify the name of the RS Database.
-        
+
         .PARAMETER IsExistingDatabase
-            Specify this switch if the database to use already exists.
-        
+            Specify this switch if the database to use already exists, and prevent creation of the database.
+
+        .PARAMETER IsExistingDatabaseUser
```

Comment:
I don't think we should modify the logins of the database on behalf of the user during configuration of RS